ruby/README.md

```diff
@@ -0,0 +1,58 @@
+# HubSpot-ruby sample Webhooks app
+
+### Note on the Data Base
+This application uses MySQL database to store the events coming from Webhooks. There is a single events table:
+```
+create table if not exists events
+(
+    id              INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    event_type      VARCHAR(255),
+    object_id       int             default null,
+    event_id        bigint          default null,
+    occurred_at     bigint          default null,
+    property_name   varchar(255)    default null,
+    property_value  varchar(255)    default null,
+    created_at      datetime        default CURRENT_TIMESTAMP
+);
+```
+Please note that event_id sent by HubSpot needs to be stored as int
+
+### Setup App
+
+Make sure you have [Docker Compose](https://docs.docker.com/compose/) and [Ngrok](https://ngrok.com/) installed.
```

Comment:
Do we need Ngrok to be installed locally? Looks like it's included into `docker-compose.yml`